libs/shared/console-shared/src/lib/enable-box/enable-box.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from '__tests__/utils/setup-jest'
+import EnableBox, { EnableBoxProps } from './enable-box'
+
+const props: EnableBoxProps = {
+  checked: true,
+  setChecked: jest.fn(),
+  name: 'test',
+  description: 'test',
+  title: 'test',
+  children: <div>test</div>,
+}
+
+describe('EnableBox', () => {
```

Comment:
We can add a test to check if the children displaying when the checkbox is checked